### PR TITLE
nsis.install change to embedded

### PR DESCRIPTION
**Pull request by TheCakeIsNaOH**
## Description

Changes nsis.install to include binary. 

Formatting and style of the install script and verification file were based on the Wireshark package, I can change if you would prefer a different style.

I also fixed the uninstall, as it was not working.

The AU get-latest versioning seems to be broken, it returns 3.05 rather than the 3.5.0 of the package. I did not touch this part.

I can do another PR for the .portable package, once this one looks good, I just don't want to have to possibly make changes twice. 

## Motivation and Context

https://github.com/mkevenaar/chocolatey-packages/issues/7

## How Has this Been Tested?

Tested in Chocolatey test environment, and on a windows 10 machine. 
AU tested on a windows 10 machine. Search and Replace and get remote files work, although the versioning seems broken.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/mkevenaar/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/nsis.install/legal/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,115 @@
+Applicable licenses
```
Comment (mkevenaar) [inline automatic/nsis.install/legal/LICENSE.txt:1]: The header needs to be different: 
```suggestion
From: <INSERT URL HERE!>

Applicable licenses
```

Comment (TheCakeIsNaOH) [reply automatic/nsis.install/legal/LICENSE.txt:5]: Done.

I thought I read some discussion about this not being required anymore if the URL is in the verification, but I can't find it.

**Comment by mkevenaar** [comment]
@TheCakeIsNaOH Thank you for your contribution!

Did you want to do the same for the .portable package before I merge this, or do you want to create a separate PR?

**Comment by TheCakeIsNaOH** [comment]
Just updated this PR with .portable

**Comment by mkevenaar** [comment]
@TheCakeIsNaOH Tank you for your contribution, your changes will be merged.

**Pull request merged**